openmetadata-service/src/test/java/org/openmetadata/service/security/policyevaluator/PolicyEvaluatorTest.java

```diff
@@ -94,8 +94,8 @@ void trimResourcePermissions() {
 
     List<ResourcePermission> rpList = List.of(rp1, rp2);
     PolicyEvaluator.trimResourcePermissions(rpList);
-    assertEqualsPermissions(expectedOp1, rpList.get(0).getPermissions());
-    assertEqualsPermissions(expectedOp2, rpList.get(1).getPermissions());
+    assertEqualPermissions(expectedOp1, rpList.get(0).getPermissions());
+    assertEqualPermissions(expectedOp2, rpList.get(1).getPermissions());
```

Comment:
should we revert this change just to keep the PR clean? Seems like a " replace all " kicked in

Reply:
I will fix and check

Reply:
In java, "assertEquials" is correct, so I revert this change.